act-rules-format/act-rules-format.bs

```diff
@@ -433,6 +435,42 @@ Even concepts like headings and images can be misunderstood. These terms could r
   </blockquote>
 </aside>
 
+<aside class=example>
+  <header>Example subjective applicability for a rule testing that elements styled as a heading use correct heading markup. This rule applicability cannot be written objectively since "styled as a heading" is subjective and depends on the context of the page.</header>
+  <blockquote>
+    <p>The rule applies to any HTML element that is styled as a heading.</p>
```

Comment:
Include definition inline to indicate these are defined terms, otherwise it does seem too broad. 
> Styled as a heading: